crates/bevy_asset/src/server/mod.rs

```diff
@@ -401,6 +401,44 @@ impl AssetServer {
         handle
     }
 
+    pub(crate) fn load_erased_with_meta_transform<'a, G: Send + Sync + 'static>(
```

Comment:
This fn is effectively a carbon copy of `load_with_meta_transform` but using `get_or_create_path_handle_erased`. I don't like the code duplication - should this be combined into one fn or left separate?

Reply:
I think just deduping the `should_load` part should be enough. The rest of it is pretty small.